WTWSMS/decisions/WTWSMS_THIS_IS_SPARTA.txt

```diff
@@ -0,0 +1,398 @@
+					trait = wroth
+					trait = powerful_voice
+					trait = fertile
+					trait = lustful
+					attributes = {
+						martial = 30
+					}
+				}
+				new_character = {
+					d_three_hundred = {
+						grant_title = PREV
+						gender_succ = true_cognatic
+					}
+				}
+				create_random_priest = {
+					random_traits = yes
+					religion = ROOT
+					culture = tsakonian
+					age = 30
+					wealth = 50
+					trait = mystic
+					trait = scholar
+					trait = zealous
+					trait = powerful_voice
+					female = no
+					attributes = {
+						martial = 13
+					}
+				}
+				create_random_soldier = {
+					random_traits = yes
+					religion = ROOT
+					culture = spartan
+					age = 18
+					wealth = 50
+					trait = winter_soldier
+					trait = powerful_voice
+					female = no
+					attributes = {
+						martial = 12
+					}
+				}
+				create_random_soldier = {
+					random_traits = yes
+					religion = ROOT
+					culture = spartan
+					trait = powerful_voice
+					female = no
+				}
+				create_random_soldier = {
+					random_traits = yes
+					religion = ROOT
+					culture = greek
+					trait = powerful_voice
+					female = no
+				}
+				create_random_soldier = {
+					random_traits = yes
+					religion = ROOT
+					culture = macedonian
+					trait = powerful_voice
+					female = no
+				}
+				create_random_soldier = {
+					random_traits = yes
+					religion = ROOT
+					culture = macedonian
+					trait = powerful_voice
+					female = no
+				}
+				create_random_soldier = {
+					random_traits = yes
+					religion = ROOT
+					culture = tsakonian
+					trait = powerful_voice
+					female = no
+				}
+				create_random_soldier = {
+					random_traits = yes
+					religion = ROOT
+					culture = spartan
+					race = ROOT
+					age = 16
+					female = yes
+					trait = strong
+					trait = fertile
+					trait = lustful
+					trait = master_seductress
+					trait = powerful_voice
+					attributes = {
+						martial = 14
+					}
+				}
+				create_random_soldier = { 
+					random_traits = yes
+					female = yes
+					trait = powerful_voice
+					trait = fertile
+				}
+				create_random_soldier = { 
+					random_traits = yes
+					female = yes
+					trait = powerful_voice
+					trait = fertile
+				}
+				create_character = { # Slave
+					random_traits = yes
+					religion = nicene
+					culture = greek
+					race = ROOT
+					age = 16
+					female = no
+					trait = eunuch
+				}
+				create_random_soldier = { # Slave
+					random_traits = yes
+					religion = nicene
+					culture = random
+					race = random
+					age = 16
+					female = no
+					trait = eunuch
+				}
+				create_character = { # Slave
+					random_traits = yes
+					religion = zoroastrian
+					culture = persian
+					race = random
+					female = no
+					trait = eunuch
+				}
+				create_random_soldier = { # Slave
+					random_traits = yes
+					religion = nicene
+					culture = greek
+					race = ROOT
+					female = no
+					trait = eunuch
+				}
```

Comment:
Wouldn't culture = ROOT make more sense? Otherwise you create Spartans when they may not exist.